plugins/CoreAdminHome/Controller.php

```diff
@@ -299,6 +299,7 @@ private function handleGeneralSettingsAdmin($view)
         $view->todayArchiveTimeToLive = $todayArchiveTimeToLive;
         $view->todayArchiveTimeToLiveDefault = Rules::getTodayArchiveTimeToLiveDefault();
         $view->enableBrowserTriggerArchiving = $enableBrowserTriggerArchiving;
+        $view->showSegmentArchiveTriggerInfo = !((bool) Config::getInstance()->General['browser_archiving_disabled_enforce']);
```

Comment:
the typecast to bool is actually redundant as the `!` already forces a typecast to bool.

Reply:
Might actually even be better to use `Rules::isBrowserArchivingAvailableForSegments()` here.

Reply:
Thanks - I agree and have updated accordingly.